### ISSUE TITLE
Can you make this extension render xml files with "atom+xml" Content-Type too?

### ISSUE THREAD

**Issue by kaushalmodi** [Sytten/PrettyXML#1]
Hello,

I discovered your "Pretty XML" Firefox extension just today as I was trying for a way to look at the Atom feed (xml) of my blog in a prettified fashion.

My site has both RSS and Atom feeds.

The RSS feed xml ( https://scripter.co/index.xml with _Content-Type: application/**xml**_) renders nicely using Pretty XML.

But when I load the Atom feed xml ( https://scripter.co/atom.xml with _Content-Type: application/**atom+xml**_ ), Pretty XML does not kick in to render that xml.

Can Pretty XML be activated for "application/atom+xml" Content-Type too?

Thanks!

**Comment by jpawlowski** [comment]
Oh, don't forget to mention the correct mime type of RSS, which is `application/rss+xml` so that should be handled nicely as well. ;-)

@kaushalmodi note that inline rendering in the browser is considered a fallback unless there is another application claiming to be able to handle that mime type. In that case, the plugin shall give an option to the end user to decide to open the file in the browser or the 3rd party application. It is likely not possible to do this in parallel but at least a configuration switch would be nice to have.

**Comment by kaushalmodi** [comment]
When I try to open that *application/atom+rss* Content-Type file (`atom.xml`), I get this unhelpful prompt on Windows 10:

![image](https://user-images.githubusercontent.com/3578197/62638562-ba20c780-b90b-11e9-8685-4d16c376dafb.png)

But when I open the *application/xml* Content-Type file (`index.xml`), Pretty XML is able to steal the opportunity to display it in-browser.

I wish it to do the same for that Atom feed too.

**Comment by kaushalmodi** [comment]
> inline rendering in the browser is considered a fallback unless there is another application claiming to be able to handle that mime type

What I am trying to say is that it's very likely that more applications would be claiming to handle *application/xml* than *application/atom+xml*, but still Pretty XML is handling *application/xml* fine.

So my best guess is that it only needs to be receptive to the other xml MIME type variants too.

**Comment by Sytten** [comment]
I don't have the time to work on it until mid-august, but I will make the necessary change. I need to do a cleanup of this repo because a lot of the code is legacy from the original author.
